test/e2e/specs/specs-calypso/wp-editor__edit-media-spec.js

```diff
@@ -24,6 +25,10 @@ describe( `[${ host }] Media: Edit Media (${ screenSize }) @parallel @jetpack`,
 	} );
 
 	it( 'Select a random media item and click edit', async function () {
+		// Makes sure that the Quick Switch modal will be dismissed.
+		const guideComponent = new GuideComponent( this.driver );
+		await guideComponent.dismiss( 1000, '.nav-unification-quick-switch-modal' );
```

Comment:
Rather that duplicating this all over the specs, can we do it only once after the login flow, similarly to what we did for the nav unification modal?

https://github.com/Automattic/wp-calypso/blob/e81e269143b14a374cd6a07c806e93fbeeeba6df/test/e2e/lib/flows/login-flow.js#L91-L93

Reply:
On never mind, just realized that the quick switch modal might not appear after the login flow, since it depends on the what the current screen is.

Reply:
This worked for the nav unification modal because it was shown on the "Home" screen.